concourse/steps/version.mako

```diff
@@ -43,6 +44,9 @@ elif version_operation in ('finalize-skip-patchlevel-zero', 'finalise-skip-patch
   version_operation_kwargs['skip_patchlevel_zero'] = True
 elif version_operation == 'noop':
   version_operation_kwargs['operation'] = 'noop'
+elif version_operation == 'inject-timestamp':
+  version_operation_kwargs['operation'] = 'set_prerelease'
+  prerelease = f'timestamp-{int(datetime.datetime.now(tz=datetime.timezone.utc).timestamp())}'
```

Comment:
I am fine w/ epoch instead of  RFC3339-timestamp. we should check w/ @rfranzke whether he is also fine w/ that, as I think he explicitly mentioned an rfc-3339-ts

Reply:
I've decided to use an epoch timestamp instead of the RFC3339 one to stay compliant with [SemVer](https://semver.org/). Also, the epoch timestamp is just as good for sorting purposes as the RFC3339 timestamp.

@rfranzke is there any other requirement which enforces RFC3339?

Reply:
I think epoch should also be fine. I was following https://fluxcd.io/flux/guides/image-update/#imagepolicy-examples (see very last example at the bottom of the page), and here they talk about `RFC3339`. However, epoch is also alphabetically-sortable, and that's all that counts, I guess.